# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mesa/hotfix-blake3.patch.x86
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

blake3_dispatch.c.o: in function `blake3_compres s_in_place':
blake3_dispatch.c:(.text.blake3_compress_in_place+0x44): undefined reference to `blake3_compress_in_place_avx512'ld: blake3_dispatch.c:(.text.blake3_compress_in_place+0x5c): undefined reference to `blake3_compress_in_place_sse41'

--- mesa-23.2.1/src/util/blake3/blake3_impl.h.vanilla	2023-10-10 19:09:03.977959798 +0200
+++ mesa-23.2.1/src/util/blake3/blake3_impl.h	2023-10-10 19:09:13.095960037 +0200
@@ -9,6 +9,11 @@
 
 #include "blake3.h"
 
+#define BLAKE3_NO_AVX512
+#define BLAKE3_NO_AVX2
+#define BLAKE3_NO_SSE41
+#define BLAKE3_NO_SSE2
+
 // internal flags
 enum blake3_flags {
   CHUNK_START         = 1 << 0,
